pkgs/by-name/md/md4c/package.nix

```diff
@@ -57,11 +62,11 @@ stdenv.mkDerivation rec {
         "Unicode"). See more details below.
       - Permissive license: MD4C is available under the MIT license.
     '';
-    homepage = "https://github.com/mity/md4c";
-    license = licenses.mit;
-    maintainers = with maintainers; [ AndersonTorres ];
+    changelog = "https://raw.githubusercontent.com/mity/md4c/${finalAttrs.src.rev}/CHANGELOG.md";
```

Comment:
may I ask why raw.githubusercontent ? over github.com ?

```
 rg changelog | grep raw.github | wc -l
26

 rg changelog | grep github.com | wc -l
4866
```

Reply:
Yes, you can :)

Nothing special, I was merely trying to read the file without GitHub's automatic formatting bells and whistles.

Reply:
Thanks for clarification. It's always better to ask first :)